pkg/ide/jetbrains.go

```diff
@@ -67,11 +68,33 @@ func downloadJetbrainsIDE(projectHostname, downloadUrl, downloadPath string) err
 
 	view_util.RenderInfoMessage(fmt.Sprintf("Downloading the IDE into the project from %s...", downloadUrl))
 
-	downloadIdeCmd := exec.Command("ssh", projectHostname, fmt.Sprintf("mkdir -p %s && curl -fsSL %s | tar -xz -C %s --strip-components=1", downloadPath, downloadUrl, downloadPath))
-	downloadIdeCmd.Stdout = io.Writer(&util.DebugLogWriter{})
+	pipeReader, pipeWriter := io.Pipe()
+
+	downloadIdeCmd := exec.Command("ssh",projectHostname, fmt.Sprintf("mkdir -p %s && wget %s -O - && tar -xz -C %s --strip-components=1", downloadPath, downloadUrl, downloadPath))
```

Comment:
@harkiratsm while the download spinner does work, this method of downloading the file and then extracting is much much slower than piping the output of `wget` directly to `tar`.

Because of this, I suggest that we use the builtin progress bar from `wget` like so:
```go
downloadIdeCmd := exec.Command("ssh", projectHostname, fmt.Sprintf("mkdir -p %s && wget -q --show-progress --progress=bar:force -pO- %s | tar -xzC %s --strip-components=1", downloadPath, downloadUrl, downloadPath))
```
and then simply attach the cmd stdout and err to the os stdout and err.

This will also remove the need for a 3rd party dependency.

Reply:
I understand.

Reply:
Here is the updated output.
 
![Screenshot from 2024-03-21 20-14-53](https://github.com/daytonaio/daytona/assets/71957674/0b325d38-48c7-4981-8877-94d3d6da9065)

Reply:
That's great! The code looks good.

I'll be holding off on approving the PR until we merge #260. After the merge, you will rebase so the action can check for lint and formatting problems. This will help us improve the review process and to keep the codebase style consistent.

Reply:
@harkiratsm one more thing. Please change the subject of your commit to `feat: ...`. `chore: ...` should be for commits that changed CI procedures or minor updates to the README, etc. For code changes, you should use `fix:`, `refactor:` and `feat:` (feature). You can read more about Conventional commits [here](https://www.conventionalcommits.org/en/v1.0.0/#:~:text=The%20commit%20contains,trailer%20format.).

Reply:
Got it :+1: